teamchains/inputs/multiple_use_invite_exceed_2.iced

```diff
@@ -0,0 +1,85 @@
+description: """Multiple use invite, used for invite links.
+
+Use the invite too many times, should fail in sigchain player.
+
+Loading the links up to 5th link (3rd change_membership) should work - adding
+"lotus" brings the invite use count to 5, which is the max_uses. Then loading
+next link - adding "rosemary" - triggers an error, becuase 6th add for this
```

Comment:
becuase -> because

Reply:
thanks, i need a spell checker

https://marketplace.visualstudio.com/items?itemName=zapu.vscode-aspell

maybe this would work